scripts/parse-vcf/README.md

```diff
@@ -0,0 +1,40 @@
+# bcftools image and annotation processing script
+
+## Environment
+
+The Dockerfile in this directory creates a bcftools image and installs some additional libraries for parsing local or remote vcf files using the `parse_vcf` python module. This parsing module is tailored to gnomAD annotation files and builds a csv comprised of the fields required by the SSMP application.
+
+To build the image, run `docker build -t bcftools .` from the directory containing the Dockerfile.
+
+The entrypoint is the bcftools binary, so you can run an arbitrary bcftools command in a local directory against a local file with something like:
+
+```bash
+docker run --rm -v `pwd`:`pwd` -w `pwd` bcftools stats <some-vcf-in-the-current-working-directory.bgz>
+```
+
+## Running the script
+
+You can use the image to run the `parse_vcf` python module, which will parse a gnomAD annotation vcf and write the filtered results to a csv that can be imported into a database.
+
+```bash
+ docker run --rm -v `pwd`:`pwd` -w `pwd` --entrypoint="python" bcftools -m parse_vcf --source gnomad_2.1.1_exome --out my-local-results-directory https://gnomad-public-us-east-1.s3.amazonaws.com/release/2.1.1/vcf/genomes/gnomad.genomes.r2.1.1.sites.vcf.bgz
```

Comment:
Do we know what gnomad version we're supposed to use going forward, or is the consensus is that we will always use the most up-to-date one?

Reply:
For the 2x releases, the most recent I think is what we'd like. By specifying the source/version in the `--source` arg, there will be a record of it for each annotation, which hopefully will cut down on confusion. The 3x releases are for GRCh38 only, so there may be special considerations there, but I would also assume the latest would be best. If a new version is released and users would like to see it, the script can be run again and the database updated.